test/unit/jest.config.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+const glob = require( 'glob' ).sync;
+
+// Finds all packages which are transpiled with Babel to force Jest to use their source code.
+const transpiledPackageNames = glob( 'packages/*/src/index.js' )
```

Comment:
I'm wondering if we ought to start codifying / documenting where we have these conventions, i.e. "sources should have a `src/index.js` file".

Reply:
I would not only document them but maybe try to go further and centralize the source of those lists. I will open a follow-up issue.

Reply:
#14508 is up for grabs :)